week-3/E-array-map/exercise.js

```diff
@@ -3,3 +3,9 @@
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
+var multiplyNumbers = numbers.map(number => {
+    return number * 100;
```

Comment:
Very well! 
The instructions also mention writing multiple solutions using different syntax - could you add those examples too?

Reply:
I'll do.